client/src/components/containers/CodeReviewMetricsContainers/CodeReviewFirstResponse/CodeReviewFirstResponse.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "src/styles/index.scss";
+
+.firstReviewResponseTimeContainer {
+  @include flex-align-center;
+  @include flex-justify-space-between;
+
+  width: 100%;
+}
+
+.firstReviewResponseTime {
+  width: 90%;
+}
+
+.timelineIconColor {
+  color: $app-quaternary-text-color;
+  cursor: pointer;
+  font-size: $icon-size-xl;
```

Comment:
Use font size variables

Reply:
as the class is for icon, i have used icon token, 

changed it to font token.